manifests/examples/customca/value_custom_ca.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  profile: demo
+  meshConfig:
+    ca:
+      # Use istiod_side to specify CA Server integrate to Istiod side or Agent side
+      istiodSide: true
```

Comment:
Just to be sure: This commit allows the external CA to integrate directly with both the Istio-agent and with Istiod, correct? If so, could you consider changing the diagram mentioned in the github "conversation" ? It seems to imply that that the only way to integrate the external CA is through Istiod

Reply:
This pull request (like above diagram) only support external CA to integrate directly with Istiod.
According MeshConfig 'istiodSide' : Because the integration server implement Istio CSR API and it's used both Istio-Agent and Istiod, so this flag makes it easier for the user to understand what happen. Besides, This flag prepare for the long-term solution (here is link to discussion: [CustomCA MeshConfig pull request](https://github.com/istio/api/pull/1596#issuecomment-678032022))